virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -9,6 +9,13 @@ Before you install {VirtProductName}, ensure that your {product-title} cluster m
 * Your cluster must be installed on
 xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure with Red Hat Enterprise Linux CoreOS workers.
 
+* Additionally, your cluster must use xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer-provisioned installation] and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[deploy machine health checks] to maintain High Availability (HA).
+
+[NOTE]
+====
+In {VirtProductName} clusters installed using installer-provisioned installation, if a node fails the MachineHealthCheck and becomes unavailable to the cluster, a `RunStrategy` determines what happens next, depending on a series of conditions. xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-about-runstrategies-vms_virt-create-vms[About RunStrategies for virtual machines] provides more detailed information about RunStrategies.
```

Comment:
* Same about it being "installer-provisioned infrastructure".
* I'd rearrange the last sentence to instead say something like "See xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-about-runstrategies-vms_virt-create-vms[About RunStrategies for virtual machines] for more detailed information about RunStrategies.